### PR TITLE
ENH - Add logo and favicon to docs

### PR DESCRIPTION
**Pull request by trallard**
> !NOTE
> I started this PR off the branch in #65, so that PR must be merged before this one

This PR:

- Moves the logo and favicon one level up in the `_static` directory so that this can be rendered by Sphinx book
- Updates the paths to the favicon and logo on the landing page
- Changes the URL to the tutorial to relative URLs (was pointing to the built RTD docs)
- Updates the contribution guidelines to point to the right source directory
- Removes some accidentally checked files (doctrees)

🔗 Link to PR preview: https://jax-ai-stack--114.org.readthedocs.build/en/114/

### CODE REVIEW

**Review by pavithraes — APPROVED** [review]
Thanks, Tania. Looks good to me. :)

**Comment by jakevdp** [comment]
What's the purpose of moving all the files here? This is going to cause conflicts with every other in-flight PR, so I'd lean toward not merging this change.

Can we add the logo and favicon without modifying the file structure of the repo?

**Comment by jakevdp** [comment]
Ah, got it. In that case we need to wait for #65 to be merged, at which point we can rebase this and the diff should be small.

**Comment by jakevdp** [comment]
OK, the style update is merged. Can you rebase this? Thanks!

**Review by jakevdp — CHANGES_REQUESTED** [review]

File: docs/source/_static/ai-stack-logo.svg
[the remark this answers is not in the document]
Comment (jakevdp) [reply docs/source/_static/ai-stack-logo.svg:1]: Also, I don't understand the need to move this out of the `images` subdirectory. Can you elaborate on the reason?

**Review by jakevdp — APPROVED** [review]
Thanks!

**Pull request merged**